Sources/KlaviyoUI/KlaviyoWebView/Development Assets/PreviewGridViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  PreviewGridViewController.swift
+//  klaviyo-swift-sdk
+//
+//  Created by Andrew Balmer on 10/23/24.
+//
+
+#if DEBUG
+import Foundation
+import UIKit
+
+/// A sample grid view with randomly-colored tiles for internal Xcode preview purposes only.
+class PreviewGridViewController: UIViewController, UICollectionViewDataSource {
+    var collectionView: UICollectionView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(0.5), heightDimension: .fractionalHeight(1.0))
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+        let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .fractionalWidth(0.4))
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+        group.interItemSpacing = .flexible(12)
+        let section = NSCollectionLayoutSection(group: group)
+        section.interGroupSpacing = 12
+        section.contentInsets = .init(top: 0, leading: 16, bottom: 0, trailing: 16)
+
+        let layout = UICollectionViewCompositionalLayout(section: section)
```

Comment:
it's been a minute since I used this!

Reply:
haha me too. I've been in SwiftUI world for so long so UIKit is a throwback for me